code/game/turfs/closed/indestructible.dm

```diff
@@ -246,16 +246,26 @@ INITIALIZE_IMMEDIATE(/turf/closed/indestructible/splashscreen)
 	smoothing_flags = NONE
 	canSmoothWith = null
 	smoothing_groups = null
+	/// What kind of turf should be visually represented under this door?
+	var/turf/floor_to_copy = /turf/open/floor/plating
```

Comment:
why does this exist?

Reply:
The space under fakedoors are black voids; otherwise, and I figured since I was already working on a bunch of maps that most prominently contained these; I may as well knock them out. This var is specifically to make subtyping easier.